app/templates/components/ui-table/cell/events/view/tickets/orders/cell-order.hbs

```diff
@@ -29,7 +29,7 @@
       <i class="check icon"></i>
     {{/ui-popup}}
   {{/if}}
-  {{#if (and (not-eq record.status 'cancelled') (not-eq record.status 'expired') (can-modify-order record))}}
+  {{#if (and (not-eq record.status 'cancelled') (not-eq record.status 'expired') (can-modify-order record) (not-eq extraRecords.status 'cancelled'))}}
```

Comment:
There is already `record.status`, Why to add `extraRecords.status` ?

Reply:
i  am not able to understand what is use of "record.status" though it's not working . if it's working then cancel order button should not be functional in that section but it is .In the code every where else  `extraRecord` is being used for getting status.

Reply:
```suggestion
  {{#if (and (not-eq extraRecords.status 'cancelled') (not-eq extraRecords.status 'expired') (can-modify-order record))}}
```
Change to this & check if it is working for expired orders too.

Reply:
Do this and check, If it is showing for cancelled then it must be showing for expired too
```suggestion
  {{#if (and (not-eq extraRecords.status 'cancelled') (not-eq extraRecords.status 'expired') (can-modify-order record))}}
```